stress_test.py

```diff
@@ -0,0 +1,390 @@
+                        '--request-docker-image',
+                        image,
+                    ]
+                )
+
+    def _test_infinite_memory(self):
+        if not self._args.test_infinite_memory:
+            return
+        self._set_worksheet('infinite_memory')
+        file = self._create_infinite_memory_script()
+        self._run_bundle([self._cl, 'upload', file.name()])
+        self._run_bundle([self._cl, 'run', ':' + file.name(), 'python ' + file.name()])
+        file.delete()
+
+    def _create_infinite_memory_script(self):
+        code = 'a=["codalab stress test memory"]\nwhile True: a.extend(a); print(a)'
+        return TestFile('stress_memory.py', content=code)
+
+    def _test_infinite_gpu(self):
+        if not self._args.test_infinite_gpu:
+            return
+        self._set_worksheet('infinite_gpu')
+        file = self._create_infinite_memory_script()
+        self._run_bundle([self._cl, 'upload', file.name()])
+        for _ in range(self._args.infinite_gpu_runs_count):
+            self._run_bundle(
+                [self._cl, 'run', ':' + file.name(), 'python ' + file.name(), '--request-gpus=1']
+            )
+        file.delete()
+
+    def _test_infinite_disk(self):
+        if not self._args.test_infinite_disk:
+            return
+        self._set_worksheet('infinite_disk')
+        # Infinitely write out random characters to disk
+        self._run_bundle([self._cl, 'run', 'dd if=/dev/zero of=1g.bin bs=1G;'])
+        self._run_bundle([self._cl, 'run', 'dd if=/dev/urandom of=/dev/sda;'])
+
+    def _test_many_disk_writes(self):
+        self._set_worksheet('many_disk_writes')
+        for _ in range(self._args.large_disk_write_count):
+            # Write out 1 GB worth of bytes out to disk
+            self._run_bundle([self._cl, 'run', 'dd if=/dev/zero of=1g.bin bs=1G count=1;'])
+
+    def _set_worksheet(self, run_name):
+        worksheet_name = self._create_worksheet_name(run_name)
+        uuid = run_command([self._cl, 'new', worksheet_name], force_subprocess=True)
+        run_command([self._cl, 'work', worksheet_name], force_subprocess=True)
+        run_command([self._cl, 'wedit', '--tag=%s' % StressTestRunner._TAG], force_subprocess=True)
+        return uuid
+
+    def _create_worksheet_name(self, base_name):
+        return 'stress_test_{}_ws_{}'.format(base_name, self._generate_random_id())
+
+    def _generate_random_id(self):
+        return ''.join(
+            random.choice(string.ascii_lowercase + string.ascii_uppercase) for _ in range(24)
+        )
+
+    def _run_bundle(self, args):
+        args.append('--tags=%s' % StressTestRunner._TAG)
+        return run_command(args, force_subprocess=True)
+
+    def cleanup(self):
+        if self._args.bypass_cleanup:
+            return
+        print('Cleaning up...')
+        bundles = run_command(
+            [self._cl, 'search', 'tags=%s' % StressTestRunner._TAG, '.limit=10000', '--uuid-only'],
```

Comment:
10000 is a lot, worried about it timing out.  I'd put this in a loop: I'd cap it at 1000 and put it in a loop.